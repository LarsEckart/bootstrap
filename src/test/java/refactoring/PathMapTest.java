package refactoring;

import org.approvaltests.Approvals;
import org.approvaltests.reporters.UseReporter;
import org.approvaltests.reporters.macosx.BeyondCompareMacReporter;
import org.junit.jupiter.api.Test;

@UseReporter(BeyondCompareMacReporter.class)
class PathMapTest {

  @Test
  void test_initialization() {
    Approvals.verify(XMLToJson.pathMap);
  }
}
