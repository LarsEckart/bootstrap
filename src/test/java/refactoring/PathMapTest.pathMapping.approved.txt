path maps


 -> //toc
fk:AMM24_fk:AMM24-FM -> //folder[@key='AMM24']/folder[@key='AMM24-FM']
dk:AMM24 -> //doc[@key='AMM24']
dt:AMM24 -> //doc[@type='AMM24']
dth:AMM24 -> //doc[@type='history'='AMM24']
dtrn:AMM24 -> //doc[@trnum='AMM24']
fk:AMM24 -> //folder[@key='AMM24']
ft:AMM24 -> //folder[@type='AMM24']
fth:AMM24 -> //folder[@type='history'='AMM24']
dk:AMM24_dt:AMM24_dth:AMM24_dtrn:AMM24_fk:AMM24_ft:AMM24_fth:AMM24 -> //doc[@key='AMM24']/doc[@type='AMM24']/doc[@type='history'='AMM24']/doc[@trnum='AMM24']/folder[@key='AMM24']/folder[@type='AMM24']/folder[@type='history'='AMM24']
unknown:AMM24 -> class java.lang.Exception no mapping found
unknown:AMM24_ft:any -> class java.lang.Exception no mapping found
